build.gradle

```diff
@@ -220,6 +226,18 @@ subprojects {
   tasks.withType(ScalaCompile) {
     scalaCompileOptions.useAnt = false
 
+    scalaCompileOptions.additionalParameters = [
+      "-deprecation",
+      "-unchecked",
+      "-encoding", "utf8",
+      "-target:jvm-1.7",
+      "-Xlog-reflective-calls",
+      "-feature",
+      "-language:postfixOps",
+      "-language:implicitConversions",
```

Comment:
I think so, there isn't anything wrong with implicit conversions as extension methods IMO (implicit value classes give you that with no performance penalty). I can remove it if people feel strongly, but the language imports seem like useless boilerplate in most cases.